projects/kalidea/kaligraphi/src/lib/02-form/kal-textarea/kal-textarea.component.ts

```diff
@@ -35,6 +37,33 @@ export class KalTextareaComponent extends FormElementComponent<string> implement
     super();
   }
 
+  private _disabled = false;
+
+  @Input()
+  @Coerce('boolean')
+  get disabled(): boolean {
+    return this._disabled;
+  }
+
+  set disabled(value: boolean) {
+    this._disabled = value;
+
+    if (this.control) {
+      this.setDisabledState(this._disabled);
+    }
+  }
+
+  /**
+   * @inheritDoc
+   */
+  setDisabledState(disabled: boolean) {
```

Comment:
set in formElement instead ( and store control created by createControlAndSubscriptions in FormElement